src/sentry/taskworker/registry.py

```diff
@@ -159,7 +159,7 @@ def send_task(self, activation: TaskActivation, wait_for_delivery: bool = False)
                 ArroyoTopic(name=topic.value),
                 KafkaPayload(key=None, value=activation.SerializeToString(), headers=[]),
             )
-
+        metrics.incr("taskworker.registry.send_task.scheduled")
```

Comment:
Could we add the namespace/task tags so that we can compare with the produce_future callback?